pi5/config.go

```diff
@@ -2,8 +2,7 @@
 package pi5
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
```

Comment:
Please us ethe go standard errors library "errors" rather than the public archive one.